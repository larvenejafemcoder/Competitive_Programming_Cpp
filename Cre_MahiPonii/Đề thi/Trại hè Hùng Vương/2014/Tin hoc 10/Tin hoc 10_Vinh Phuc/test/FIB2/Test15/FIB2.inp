21
27 109391
9 52
0 1
28 265365
6 4
40 90728948
40 3906463
14 592
39 55877738
12 87
12 225
15 939
35 7440468
36 22613077
26 7373
32 1270121
23 42138
3 1
40 2959087
31 604622
43 257682296
