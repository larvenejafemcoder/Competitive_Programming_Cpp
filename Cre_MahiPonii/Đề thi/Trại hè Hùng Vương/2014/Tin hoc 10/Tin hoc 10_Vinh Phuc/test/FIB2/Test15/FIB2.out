41784
20
1
101361
2
34655374
1492136
226
21343397
33
86
359
2842006
8637427
2816
485143
16095
0
1130271
230945
98425879
